axelor-human-resource/src/main/resources/views/ProjectPlanningTime.xml

```diff
@@ -108,6 +110,11 @@
 		<attribute name="value" for="task" expr="eval:_task"/>
 		<attribute name="value" for="project" expr="eval:_project"/>
 		<attribute name="value" for="userSet" expr="eval:_project.membersUserSet" if="_project != null"/>
+		
+		<attribute name="value" for="date" expr="#{__date__}" if="_project == null"/>
+		<attribute name="value" for="user" expr="eval:__user__" if="_project == null &amp;&amp; __user__ in __repo__(Project).find(_parent.project.id).membersUserSet"/>
+		<attribute name="value" for="project" expr="eval:_parent?.project" if="_project == null"/>
+		<attribute name="value" for="task" expr="eval:_parent" if="_project == null &amp;&amp; _task == null"/>
```

Comment:
add condition that parent model = task model.  for both value "project" and "task"